bin/_info

```diff
@@ -47,6 +47,8 @@ def main() -> int:
     print('others')
     print()
     with _console():
+        _call('pypy3', '--version', '--version')
+        print()
```

Comment:
I would put this after the other pythons -- just because most are going to be looking for cpython versions first